__tests__/integration/mocks/node_modules/@gov.au/_test-00/lib/sass/_globals.scss

```diff
@@ -1,4 +1,4 @@
-/*! @gov.au/core v1.0.0 */
+/*! @gov.au/_test-00 v1.0.1 */
```

Comment:
Why is the core included in git? I see this is the `_test-00` mock file but it shouldn't have any sass in it no?

Reply:
This is a test for a module with sass that doesn't compile to any css. Maybe this is something that should be fixed.